jetson/db/config.py

```diff
@@ -0,0 +1,8 @@
+SQL_HOST = ''
+USER_NAME = ''
+PASSWORD = ''
+KEYSPACE = ''
+
+FTPHOST = ''
+FTPUSER = ''
+FTPPASS = ''

```

Comment:
Why is there an empty `config.py` file? if this is user specific it should be added to the `.gitignore` and kept locally. if this file also just contains values then it shouldn't be a `.py` file it should be a `json` file or similar config file format like `xml`.

Reply:
I've been using [params.py](https://github.com/PurdueCAM2Project/Embedded2/blob/master/jetson/goggles/params.py ) for hyperparameters for training the classifier, but it's just a python dict. 
1. Because it's user-specific should it also be in the .gitignore (and should I add a warning in the classifier script if the file is not there)?
2. Should I make params.py into a json or xml? What would be the benefit of that?

Reply:
So user specific config files or non functional files shouldn't be source files. This isn't as clear with python since it's interpreted but imagine this is C or C++ every user would need to build their own version of the project for it to work. That is poor design from the perspective of deployability.

I recommend `json` as the config file of choice because it provides a simple key value pair structure, is easily sent over internet connections, and pythons native `json` library can parse the file right into a python `dict` which it sounds like you were already using.

Reply:
Good to know. The hyperparameters (and associated classifier script) probably won't be part of deployment but I'll convert it to a `json`.